modules/v2x/v2x_proxy/obu_interface/grpc_interface/grpc_client.h

```diff
@@ -43,21 +48,14 @@ class GrpcClientImpl {
   bool InitFlag() { return init_flag_; }
 
   /*function that send car status msg through grpc
-  @param input carstatus type msg shared ptr
-  */
-  void SendMsgToGrpc(const std::shared_ptr<CarStatus> &msg);
-
-  /*function that send perception obstacles msg through grpc
-  @param input perception obstacles type msg shared ptr
+  @param input car_status type msg shared ptr
   */
-  void SendMsgToGrpc(
-      const std::shared_ptr<apollo::perception::PerceptionObstacles> &msg);
+  void SendMsgToGrpc(const std::shared_ptr<::apollo::v2x::CarStatus> &msg);
 
  private:
   //  grpc service stub
-  std::unique_ptr<CarToObu::Stub> stub_;
-  int carstatus_tv_nsec_ = 0;
-  int perception_tv_nsec_ = 0;
+  std::unique_ptr<::apollo::v2x::CarToObu::Stub> stub_;
+  int car_status_tv_nsec_ = 0;
```

Comment:
Please make sure `int` is enough to hold `tv_nsec` values.

Reply:
Yes. It will not greater than 500,000,000